telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
```

Comment:
That's true, would we need a hardware component for that, or can that be implemented in the firmware (or is there another board we can read a more reliable version of the timestamp from)?